internal/utils.py

```diff
@@ -90,3 +91,28 @@ def create_multi_messages(num_msgs, file_name):
         file_data["body"]["messages"] = messages
         file.seek(0)
         json.dump(file_data, file, indent=4)
+
+# `execute_tx_by_type` is used to execute the tx by type.
+def execute_tx_by_type(command, unsigned=False, sequence=None):
```

Comment:
Do we need this as a separate function? We can use old one directly. instead of taking unsigned, sequence here, append it before calling this function itself.

Reply:
We are repeating almost same lines of code in txs related queries, so thought that can be avoid with this function.

Reply:
I don't see any usage of this with sequence and generate-only mode. We can remove this right?